### PR TITLE
fixed spell check

### PR DESCRIPTION
**Pull request by fidan93**
Fixed wrong spell of club Catlin
Added authorization check for members update controller, to be accessed only by Admin or Club director

### CODE REVIEW

File: src/main/java/com/lambdaschool/oktafoundation/SeedData.java [hunk 1]
```diff
@@ -144,7 +144,7 @@ public void run(String[] args) throws
         c1.getUsers().add(new ClubUsers(c1, userService.findByName("llama003@maildrop.cc")));
 
         //Caitlin has Arts, Basketball, Homework,
-        Club c2 = new Club("Caitlin");
+        Club c2 = new Club("Catlin");
```
Comment (Shawn2776) [inline src/main/java/com/lambdaschool/oktafoundation/SeedData.java:147]: Looks like a simple fix. Good job

File: src/main/java/com/lambdaschool/oktafoundation/SeedData.java [hunk 2]
```diff
@@ -275,11 +275,6 @@ public void run(String[] args) throws
                 "1F61E",    //  1 //2
                 "1F62D",    //  1 //1
 
-
```
Comment (Shawn2776) [inline src/main/java/com/lambdaschool/oktafoundation/SeedData.java:278]: Good job removing extra space

**Review by Shawn2776 — APPROVED** [review]
Corrected a spelling error and removed extra space. Commit approved.

**Comment by robertmasters** [comment]
simple and to the point, good job.

**Review by robertmasters — APPROVED** [review]

**Pull request merged**